### ISSUE TITLE
Get Image name of running container

### ISSUE THREAD

**Issue by lorenzo93** [mariotoffia/FluentDocker#56]
Hi,

I am listing the available containers with `_docker.GetImages()`, putting the result into a list and casting each element to DockerContainerService.

How it is possible from an object like that to have the Image name which the image was created from?

**Comment by mariotoffia** [comment]
@lorenzo93 I've committed (but not released) where you can get back layers for the image when doing ```GetImages()```. I'll do a release this week with this included. Please see this how to resolve parent images (https://github.com/moby/moby/issues/22140). I will save this issue to create a function to actually iterate the layers and images. If you want, you can do a PR for this.

```cs
var config = Host.GetImages().First(x => x.Name == "postgres").GetConfiguration(fresh: true);
// Fetches the image configuration including RootFS Layers that can be found in other parent images
```

Cheers,
  Mario

**Comment by mariotoffia** [comment]
@lorenzo93 Later on I'll add a Image property on a running container so you don't need to do this yourself.

**Comment by mariotoffia** [comment]
@lorenzo93 I've added a property on the ```IContainerService``` called ```Image```. It reflects the image of the current runtime container. It is however in a feature branch and will hopefully be released this weekend.

**Comment by mariotoffia** [comment]
It is now published as 2.4.0 so I'll close this issue.
